src/button-dropdown/category-elements/expandable-category-element.tsx

```diff
@@ -90,7 +90,12 @@ const ExpandableCategoryElement = ({
               },
             } as GeneratedAnalyticsMetadataButtonDropdownExpand | GeneratedAnalyticsMetadataButtonDropdownCollapse)
       )}
+
+      //Add Rendering of icon image in expandable button
     >
+      {(item.iconName || item.iconUrl || item.iconSvg) && (
+        <InternalIcon name={item.iconName} url={item.iconUrl} svg={item.iconSvg} alt={item.iconAlt} />
```

Comment:
Remember that there is a missing space between this icon and the text. Same in the mobile version below.